pyigm/clustering/xcorr_utils.py

```diff
@@ -257,42 +259,39 @@ def random_gal(galreal, Nrand, Nmin=20, DZ=0.01, smooth_scale=10.,
        Maximum magnitude for the binning (faint end)
     delta_mag : float, optional
        Step size for magnitude binning
+    delta_mag : float (optional)
+    debug : bool (optional)
 
     Returns
     -------
-    galrand : np rec array
-      Copy of galreal, Nrand times
+    VALS : dict
```

Comment:
Describe VALS: the actual histograms used to get the Cubic Spline (I think)

Reply:
added